sr_hand_config/package.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>sr_hand_config</name>
+  <version>0.0.0</version>
+  <description>The sr_hand_config package</description>
+  <maintainer email="software@shadowrobot.com">user</maintainer>
```

Comment:
Change user with The Shadow Robot Software Team. Also you can add author and add your name